Tests/Clappr_Tests/Classes/Mocks/PlayerMock.swift

```diff
@@ -2,11 +2,16 @@ import AVFoundation
 
 class PlayerMock: AVPlayer {
     private var accessLogEvent: AccessLogEventMock
-
-    init(accessLogEvent: AccessLogEventMock) {
+    private var isFinished: Bool
+    private var _playerItem: PlayerItemMock
```

Comment:
Can we use other name to this variable since it is not common on Swift Guidelines to use underscore in private variables?

Reply:
Done ;)